### PR TITLE
detail page loading bug

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: ui/src/conferences/containers/DetailPageContainer.jsx [hunk 1]
```diff
@@ -23,25 +23,11 @@ import { newSearch } from '../../actions/search';
...
-  useEffect(
-    () => {
-      dispatch(fetchConference(recordId));
-      dispatch(newSearch(CONFERENCE_CONTRIBUTIONS_NS));
-      window.scrollTo(0, 0);
-    },
-    [dispatch, recordId]
-  );
-
+function DetailPage({ record }) {
```
Comment (drjova) [inline ui/src/conferences/containers/DetailPageContainer.jsx]: what about the ``scrollTo`` stuff?

File: ui/src/common/withRouteDataFetcher.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router-dom';
+
+import { getWrapperComponentDisplayName } from './utils';
+import LoadingOrChildren from './components/LoadingOrChildren';
+
+export default function withRouteDataFetcher(
```
Comment (drjova) [inline ui/src/common/withRouteDataFetcher.jsx]: the name is a bit confusing tbh, wouldn't be more obvious something like ``withLocationParamsDataFetcher`` or similar?

Comment (harunurhan) [reply ui/src/common/withRouteDataFetcher.jsx]: this is used to `fetch` for `data` dependencies of current `route`. So name made sense 😄

File: ui/src/conferences/containers/DetailPageContainer.jsx (hunk 1 above)
Comment (harunurhan) [reply ui/src/conferences/containers/DetailPageContainer.jsx]: this is in my notes, only thing missing in the PR right now. I think it's safe to `scrollTo(0)`, whenever pathname changes, what do you think?

File: ui/src/common/withRouteDataFetcher.jsx (hunk 2 above)
Comment (harunurhan) [reply ui/src/common/withRouteDataFetcher.jsx]: To be honest I am more concern about the fact that some components are dispatching actions which doesn't even use params, so it looks little awkward.

**Pull request merged**